pandas/core/arrays/categorical.py

```diff
@@ -1255,7 +1255,7 @@ def shape(self):
 
         return tuple([len(self._codes)])
 
-    def shift(self, periods):
+    def shift(self, periods, fill_value=None):
```

Comment:
Add `fill_value` to the docstring (with a versionadded).

Reply:
added